scripts/push-schema-changes.js

```diff
@@ -3,11 +3,19 @@
 const { updateRepo } = require("@artsy/update-repo")
 const { execSync } = require("child_process")
 const path = require("path")
+const { buildSchema, introspectionQuery, graphqlSync } = require("graphql")
+const { readFileSync, writeFileSync } = require("fs")
 
 /**
- * @param {'eigen' | 'force'} repo
+ * @param {string} repo - Name of artsy repo to update
+ * @param {string} [dest=data/schema.graphql] - Path to schema file
+ * @param {boolean} [relay=true] - Runs relay-compiler on schema file (otherwise writes JSON)
  */
-async function updateSchemaFile(repo) {
+async function updateSchemaFile(
+  repo,
+  dest = "data/schema.graphql",
+  relay = true
```

Comment:
Since we're now using multiple arguments this would probably benefit from named args so that the invocation is more readable. Something like:

```js
async function updateSchema({
  repo,
  dest = "data/schema.graphql",
  relay = true
}) {
  ...
}
```
Then down below we can do 
```js
updateSchemaFile({
  repo: "eigen",
  dest: "...",
  relay: false
})
```